HW_2/main.py

```diff
@@ -0,0 +1,38 @@
+from flask import Flask
+import random
+import string
+import pandas as pd
+
+app = Flask(__name__)
+
+
+@app.route("/generate_password")
+def generate_password(min_length = 10, max_length = 20):
```

Comment:
upper, digit and punctuation is required you should have it in your password, and in your password sometimes you don't have it.